pdc/apps/release/filters.py

```diff
@@ -39,6 +39,7 @@ class ReleaseFilter(django_filters.FilterSet):
     name = filters.MultiValueFilter(name='name')
     short = filters.MultiValueFilter(name='short')
     version = filters.MultiValueFilter(name='version')
+    sigkey = filters.MultiValueFilter(name='sigkey__key_id')
```

Comment:
Shouldn't "sigkey" be added to `fields` below?

Reply:
I have discussed with @larry-shi , as this "sigkey" is also used by compose, we don't need to put the related release info or keys in "sigkey" table (The api "/sigkey" has already created before), just put this in release's files.

Reply:
> there is an unanswered question from @larry-shi on PDC-1984. Any input on that?

For this sigkey part, I have also discussed with @larry-shi and @simozhan , no need to give a delete operation on the "sigkeys" because the data doesn't increase to much as it just includes the sigkey info. We just keep all the sigkey data here, and give an update operation.

And for release part, I should ask @simozhan for the answer.

Reply:
@hluk Add the "sigkey" in Meta fields, please help to review.

Reply:
@hluk 
But no error without adding this key in fields during my test before.
I just try to remove the fields param to test here, filters still can work, interesting.